app/serializers/event_serializer.rb

```diff
@@ -1,4 +1,6 @@
 class EventSerializer < ActiveModel::Serializer
   attributes :id, :name, :description, :image, :url, :date, :location, :instructions, :contact, :campaign_id
 
+  belongs_to :campaign
+  has_many :tasks
```

Comment:
It does work, but it is probably just getting all the fields. But yeah you are right we should have a serializer to control what we display, will add.